vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -395,6 +411,17 @@ public Builder setLogLevel(LogLevel level) {
             return this;
         }
 
+        public Builder setLocales(List<Locale> locales) {
+            this.mLocales = locales;
```

Comment:
`this.` is not needed due to use of hungarian notation.

Reply:
I added the `this.` since the rest of the methods also use `this.`

Reply:
no need to change all of them